src/bitcoin-cli.cpp

```diff
@@ -190,6 +190,15 @@ int CommandLineRPC(int argc, char *argv[])
                         throw CConnectionFailed("server in warmup");
                     strPrint = "error: " + error.write();
```

Comment:
Maybe not print the whole object, when we print the nicely formatted error?

What has always held me back from doing this (sensible) thing is that scripts may be relying on bitcoin-cli's output for errors to be as it is.